### ISSUE TITLE
Add option to also push to the users fork

### ISSUE THREAD

**Issue by mathias-baumann-sociomantic** [sociomantic-tsunami/neptune#47]
Feature request by gavin

**Comment by mihails-strasuns-sociomantic** [comment]
Push what?

**Comment by nemanja-boric-sociomantic** [comment]
Updated branches.

**Comment by mihails-strasuns-sociomantic** [comment]
To be honest that sounds more like a job for `git hub`, i.e. `git hub sync` and not something directly related to releasing.

**Comment by leandro-lucarella-sociomantic** [comment]
No, this is specific to the things that were touched by the tool, not as a general sync for all branches. I agree that this is not **needed** for releasing, but if it's convenient for users I don't see why not adding an **option** to do so.

**Comment by leandro-lucarella-sociomantic** [comment]
(it should be trivial too)

**Comment by mihails-strasuns-sociomantic** [comment]
Yeah I simply don't understand why one would want to immediately sync release branches but not anything else. I personally don't have most of release branches in my `origin` at all - what good they do there?

**Comment by mathias-baumann-sociomantic** [comment]
Setting it to blocked until you guys agreed on something
